packages/public/client/src/feature-prototypes/donation-profiles/active-authors.ts

```diff
@@ -0,0 +1,56 @@
+export const activeAuthors = [
+  '9',
+  '146133',
```

Comment:
Are these up-to-date? https://de.serlo.org/user/profile/9 doesn't look right, for example.